.github/workflows/compile_windows_binaries.yml

```diff
@@ -0,0 +1,39 @@
+name: Compile binaries for Windows
+
+on:  [pull_request, release]
+
+jobs:
+  deploy:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ['3.8' , '3.9' ]
+        numpy_version: ['1.19.1', '1.20.2', '1.21.4', '1.22.1']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set Python environment
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools
+        pip install -r requirements.txt
+        pip install wheel twine
+        pip install numpy==${{ matrix.numpy_version }}
+
+    - name: Compile library
+      run: |
+        cd aequilibrae/paths
+        python setup_Assignment.py build_ext --inplace
+        cd ..
+        cd ..
+
+    - name: Stores artifacts along with the workflow result
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ matrix.numpy_version }}
+        path: aequilibrae/paths/*.pyd
```

Comment:
I like the fail if files missing - but by specifying *.pyd - this will pass as long as ANY pyd are found.
Is there a way to specify explicitly which files are expected?